46.md

```diff
@@ -96,4 +117,10 @@ The signer key will always be the key of the user who controls the signer device
 - **ping**
   - params: []
   - result: `"pong"`
+- **create_account**
```

Comment:
my main objection to this call is that it allows the creation of a user private key without the opportunity to force the user to back up their private key. additionally, if a user did want to back up their private key their is no clear way to direct them to do so. for small instances of nsecbunker this may not be too big a deal, but it does open the door to larger service providers owning user's keys.

also `username` and `domain` params both need validation on nsecbunker side and abstracting them into client fields feels confusing for the user when either are invalid.

how would you prevent abuse of this method? could easily be used to create numerous spam accounts and NIP05 identifiers.

generally I think key generation is important to highlight to users and it should be explicit which entities a user is delegating power over their keys to. this method would create a protocol approved way of obfuscating that, though I do agree it would likely be more convenient for the user.